zerver/views/auth.py

```diff
@@ -562,9 +565,9 @@ def remote_user_sso(
     return login_or_register_remote_user(request, result)
 
 
-@has_request_variables
+@typed_endpoint
 def get_email_and_realm_from_jwt_authentication_request(
-    request: HttpRequest, json_web_token: str
+    request: HttpRequest, *, json_web_token: str
```

Comment:
This function should not have `@typed_endpoint` on it; it doesn't parse parameters from the API. It was probably a bug it had `@has_request_variables`, but you can see it was inactive since `REQ` was not used.